static/sass/components/_email-subscribe.scss

```diff
@@ -2,24 +2,20 @@
 // ==============================
 
 .odd-input-group {
-  [data-region='footer'] & {
-    align-items: flex-end;
-    display: flex;
+  align-items: flex-end;
+  display: flex;
 
-    [data-form-field] {
-      flex: 1 1 auto;
-      margin-bottom: 0;
-    }
+  [data-module='rst-body'] & {
+    margin: size('gutter') 0;
+  }
 
-    // submit btn container
-    .odd-submit-btn {
-      margin-left: size('shim');
+  [data-form-field] {
+    flex: auto;
+    margin-bottom: 0;
+  }
 
-      .small-btn {
-        font-size: size('small');
-        line-height: normal;
-        padding: 0.25rem 0.5rem;
```

Comment:
and by removing the padding here (and using the half-shim shim in the buttons.scss file, the input and buttons aren't the same height in the footer now... but just by a few pixels. I wonder if it makes more sense to use half-shim/shim on the input so this isn't an issue.